tests/TestCase/AuthenticationServiceTest.php

```diff
@@ -12,7 +12,7 @@
  * @since         1.0.0
  * @license       https://opensource.org/licenses/mit-license.php MIT License
  */
-namespace Authentication\Test\TestCase;
+namespace Authentication\Test\TestCase\Authenticator;
```

Comment:
This seems to be done by mistake. Existing namespace is already in accordance with PSR-4.

Reply:
The other way around: The subfolder should be created, and this put into it.
There shouldnt be tests directly in TestCase IMO, especially not when the class testes is not in src/ but src/Authenticator/.

Reply:
This file `AuthenticationServiceTest.php` is for `AuthenticationService.php` which is directly under `src` and hence it's in correct location directly under `TestCase` folder.